packages/react-vapor/src/components/numericInput/numeric-input.scss

```diff
@@ -4,29 +4,29 @@
 $numeric-input-height: 26px;
 $numeric-input-buttons-border-radius: 4px;
 
-.numericInput {
-    height: $numeric-input-height;
-    padding: $spacing;
-    overflow: hidden;
-    color: $medium-blue;
-    font-size: $title-font-size;
+:global(.coveo-styleguide) {
+    .numericInput {
+        height: $numeric-input-height;
+        padding: $spacing;
+        overflow: hidden;
+        color: $medium-blue;
+        font-size: $title-font-size;
 
-    text-overflow: ellipsis;
-    border: $default-border;
-    border-radius: $border-radius;
-    box-shadow: inset 0 0 3px 0 rgba($black, $transparency-5);
-}
+        text-overflow: ellipsis;
+        border: $default-border;
+        border-radius: $border-radius;
+        box-shadow: inset 0 0 3px 0 rgba($black, $transparency-5);
+    }
 
-.numericInputButton {
-    width: $numeric-input-height;
-    height: $numeric-input-height;
-    color: $pure-white;
-    line-height: $numeric-input-height - 2 * $default-border-size;
-    background-color: $blue-purple-2;
-    border-radius: $numeric-input-buttons-border-radius;
-}
+    .numericInputButton {
+        width: $numeric-input-height;
+        height: $numeric-input-height;
+        color: $pure-white;
+        line-height: $numeric-input-height - 2 * $default-border-size;
+        background-color: $blue-purple-2;
+        border-radius: $numeric-input-buttons-border-radius;
+    }
 
-:global(.coveo-styleguide) {
     .numericInputButton {

```

Comment:
It seems that we have two `.numericInputButton` in `:global(.coveo-styleguide)`?